lib/tty/pager/basic.rb

```diff
@@ -43,50 +45,71 @@ def default_prompt
       #
       # @api public
       def page(text, &callback)
-        page_num = 1
-        leftover = []
-        lines_left = @height
+        write(text, &callback)
```

Comment:
I like the idea of 'preserving' current API. My preference for any updates is to keep the current API working as much as possible.